charts/kamus/templates/deployment-encryptor.yaml

```
@@ -32,7 +32,7 @@ spec:
         release: {{ .Release.Name }}
     spec:
       serviceAccountName: {{ template "kamus.name" . }}
-      automountServiceAccountToken: false
+      automountServiceAccountToken: true
```

Maybe make it optional?
```suggestion
      automountServiceAccountToken: {{ .Values.automountServiceAccountToken}}
```

And set the default value to false?

---

That makes more sense. Is there anything else we'd need to add to the pr to register the value in the template?

---

No, just add it to the readme please

---

Updated the PR. Thanks